Source/pack.cpp

```diff
@@ -125,16 +125,15 @@ void __fastcall VerifyGoldSeeds(PlayerStruct *pPlayer)
 	}
 }
 
-void __fastcall UnPackPlayer(PkPlayerStruct *pPack, int pnum, BOOLEAN killok)
+void __fastcall UnPackPlayer(PkPlayerStruct *pPack, int pnum, BOOL killok)
 {
 	PlayerStruct *pPlayer; // esi
-	signed int v6;         // eax
 	int i;                 // [esp+10h] [ebp-8h]
 	ItemStruct *pi;        // [esp+14h] [ebp-4h]
 	PkItemStruct *pki;     // [esp+20h] [ebp+8h]

```

Comment:
Don't forget to remove the de-compiler comments, they don't have any value when the function has been cleaned.

Reply:
done